### PR TITLE
Fix node connection on startup if L1 node is not synced

### PR DESCRIPTION
**Pull request by muXxer**
We had to move parts of the init logic to the `Run` function of the node connection.

Otherwise the node connection would loop forever if L1 was not sync on startup, because the stream to update the L1 node status was not started yet, and it was only fetched once to get the initial value.

### CODE REVIEW

**Review by jorgemmsilva — APPROVED** [review]
did you run the cluster tests?

There could be some race condition where the protocol parameters are needed before those are set

**Comment by muXxer** [comment]
Thats why I left it as a draft, until the cluster tests on my PC are done :trollface:

File: packages/parameters/l1parameters.go [hunk 1]
```diff
@@ -80,7 +80,7 @@ func L1() *L1Params {
 func L1NoLock() *L1Params {
 	if l1Params == nil {
 		if isTestContext() {
-			l1Params = L1ForTesting
+			l1Params = l1ForTesting
```
Comment (jorgemmsilva) [inline packages/parameters/l1parameters.go:83]: shouldn't this use the parameters from `testutil`?

**Review by jorgemmsilva — APPROVED** [review]

File: packages/parameters/l1parameters.go (hunk 1 above)
Comment (muXxer) [reply packages/parameters/l1parameters.go:83]: Yeah, but that would create a circular dependency. I will remove this stuff in another refactor PR.
Just wanted to split this up a bit to fix the bug for now before doing a big refactor.

**Pull request merged**